Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -2436,7 +2436,8 @@ bool RenderOpenGL::MoveTextureToDevice(Texture *texture) {
         // native_format == IMAGE_FORMAT_A1R5G5B5 ? GL_RGBA : GL_RGB;
 
     unsigned __int8 *pixels = nullptr;
-    if (native_format == IMAGE_FORMAT_R5G6B5 || native_format == IMAGE_FORMAT_A1R5G5B5 || native_format == IMAGE_FORMAT_A8R8G8B8 || native_format == IMAGE_FORMAT_R8G8B8A8) {
+    if (native_format == IMAGE_FORMAT_R5G6B5 || native_format == IMAGE_FORMAT_A1R5G5B5 || native_format == IMAGE_FORMAT_A8R8G8B8 || native_format == IMAGE_FORMAT_R8G8B8A8
+         || native_format == IMAGE_FORMAT_R8G8B8) {
```

Comment:
Huh, I have a different fix for this one https://github.com/captainurist/world-of-might-and-magic/commit/8ad810fe3a1ee363d37eb8eb50e57521f19e3171

Reply:
Prefer consistent use of ARGB internally